pom.xml

```diff
@@ -229,6 +229,12 @@
                 <artifactId>javax.jms-api</artifactId>
                 <version>${javax.jms-api.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.apache.axis2</groupId>
+                <artifactId>addressing</artifactId>
+                <type>mar</type>
+                <version>${addressing.version}</version>
```

Comment:
Shall we define this as axis2.wso2.version directly, since it is coming from there.

Reply:
done